bzt/modules/blazemeter.py

```diff
@@ -1481,6 +1484,43 @@ def post_process(self):
                 for error in session.get("errors", ()):
                     raise TaurusException(to_json(error))
 
+            # if we have captured HARs, let's download them
+            self.log.info("Services: %s", self.engine.services)
+            for service in self.engine.config.get(Service.SERV):
+                # not good to reproduce what is done inside engine
+                # but no good way to get knowledge of the service in config
+                if not isinstance(service, dict):
+                    service = {"module": service}
+                module = self.engine.instantiate_module(service.get('module', ValueError()))
```

Comment:
might be better would be 
TaurusConfigError('Module name not found in service "%s"', service)